packages/runtime/src/style.tsx

```diff
@@ -24,38 +24,35 @@ export default function Style(props: StyleProps) {
     props.children.forEach(analyzeCssInDev);
   }
 
-  const sheets = props.children;
-
-  if (!sheets.length) {
-    return null;
-  }
-
-  if (isNodeEnvironment()) {
-    const filteredSheets = sheets.filter((sheet) => {
-      if (inserted[sheet]) {
-        return false;
-      }
+  const sheets = props.children.filter((sheet) => {
+    if (inserted[sheet]) {
+      return false;
+    }
 
-      inserted[sheet] = true;
+    inserted[sheet] = true;
 
-      return true;
-    });
+    return true;
+  });
 
-    if (filteredSheets.length) {
-      const rulesGroupedByBucket = groupByBucket(filteredSheets);
+  if (sheets.length) {
+    if (isNodeEnvironment()) {
+      const rulesGroupedByBucket = groupByBucket(sheets);
 
       return buckets
         .filter((bucket) => !!rulesGroupedByBucket[bucket])
         .map((bucket) => (
-          <style key={bucket} {...{ [getCompiledAttr(bucket)]: '' }} nonce={props.nonce}>
+          <style
+            key={bucket}
+            {...(bucket && { [getCompiledAttr(bucket)]: '' })}
```

Comment:
nit: we could also change this so instead of an attr we just use the bucket name?

```
<style data-c={bucket} /> 
```

Reply:
ah cool. This is what I was about to say when you asked **for the point of using es6 vs. es5 - we probably want to stay targeting es5 for now but leaving esm.. what do you think** 😅
That we might need es6 for object spread and can we do something like this. This is not a nit 😅. Its a change that should be in place hehe